### PR TITLE
[CWS] add busybox support

### PR DESCRIPTION
**Pull request by safchain**
### What does this PR do?

This PR handle busybox related execution. It uses Argv0 with an operator override to check the actual command executed.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by paulcacheux — APPROVED** [review]

File: pkg/security/secl/model/oo_symlink.go [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+			return (*Event)(ctx.Object).Exec.Process.SymlinkBasenameStr
+		},
+	}
+
+	// ProcessSymlinkPathname handles symlink for process enrtries
+	ProcessSymlinkPathname = &eval.OpOverrides{
+		StringEquals: func(a *eval.StringEvaluator, b *eval.StringEvaluator, state *eval.State) (*eval.BoolEvaluator, error) {
+			path, err := eval.GlobCmp.StringEquals(a, b, state)
+			if err != nil {
+				fmt.Printf("11111111\n")
```
Comment (Gui774ume) [inline pkg/security/secl/model/oo_symlink.go:47]: Debug log ? 😄

File: pkg/security/secl/model/oo_symlink.go [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+				return nil, err
+			}
+
+			// currently only override exec events
+			if a.Field == "exec.file.path" {
+				se1, err := eval.GlobCmp.StringEquals(symlinkPathnameEvaluators[0], b, state)
+				if err != nil {
+					fmt.Printf("22222222\n")
```
Comment (Gui774ume) [inline pkg/security/secl/model/oo_symlink.go:55]: same

File: pkg/security/secl/model/oo_symlink.go [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+				or, err := eval.Or(se1, se2, state)
+				if err != nil {
+					return nil, err
+				}
+
+				return eval.Or(path, or, state)
+			} else if b.Field == "exec.file.path" {
+				se1, err := eval.GlobCmp.StringEquals(symlinkPathnameEvaluators[0], a, state)
+				if err != nil {
+					fmt.Printf("333333333\n")
```
Comment (Gui774ume) [inline pkg/security/secl/model/oo_symlink.go:71]: 🎅🏻

**Review by Gui774ume — APPROVED** [review]
Small typos but lgtm

File: pkg/security/secl/model/oo_symlink.go (hunk 1 above)
Comment (safchain) [reply pkg/security/secl/model/oo_symlink.go]: no, I was testing the reviewers :)

**Pull request merged**